src/Simulation/Simulators.Tests/Circuits/UnitTests.qs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Simulation.Tests.QsUnitTests {
+
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Diagnostics;
+    
+    @TestOperation("QuantumSimulator")
+    operation QsharpUnitTest() : Unit {
+		Message("Worked!");
```

Comment:
How do you know if this unittest was actually executed? It could not be executed and then nothing bad would happen.

Reply:
If the test was not run, then whatever tool is being used to run tests would show that the test was not run. Maybe I'm missing the point of your comment?